client/ide/lib/index.coffee

```diff
@@ -549,52 +533,46 @@ class IDEAppController extends AppController
 
   createInitialView: (withFakeViews) ->
```

Comment:
Seems like this method became too big. Let's separate `if/else` section into separate methods.